<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="addItemPB">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>110</y>
      <width>80</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Add Item</string>
    </property>
   </widget>
   <widget class="QWidget" name="formLayoutWidget">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>30</y>
      <width>160</width>
      <height>80</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="iDLabel">
       <property name="text">
        <string>ID</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="iDLineEdit"/>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="descriptionLineEdit"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="locationLabel">
       <property name="text">
        <string>Location</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="locationLineEdit"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="descriptionLabel">
       <property name="text">
        <string>Description</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="findByIDPB">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>190</y>
      <width>80</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Find By ID</string>
    </property>
   </widget>
   <widget class="QPushButton" name="removeByIDPB">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>220</y>
      <width>80</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Remove By ID</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="findByIDLE">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>190</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="removeByIDLE">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>220</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="sortedItemsTE">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>30</y>
      <width>361</width>
      <height>121</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>10</y>
      <width>151</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Items sorted by description</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="ErrorDIalogueTE">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>280</y>
      <width>361</width>
      <height>31</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>260</y>
      <width>151</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Error Dialogue</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="foundAtTE">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>190</y>
      <width>361</width>
      <height>51</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="locationLabel_2">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>170</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Found At/Removed :</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>17</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMenu">
    <property name="title">
     <string>Menu</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuMenu"/>
   <addaction name="menuFile"/>
   <addaction name="menuHelp"/>
  </widget>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
